packages/state-manager/src/sagas/messages/verifyMessage/verifyMessages.saga.ts

```diff
@@ -1,14 +1,50 @@
 import { type PayloadAction } from '@reduxjs/toolkit'
-import { put } from 'typed-redux-saga'
+import { select, call, put, delay } from 'typed-redux-saga'
 import { messagesActions } from '../messages.slice'
-import { type MessageVerificationStatus } from '@quiet/types'
+import { MessageType, userJoinedMessage, type MessageVerificationStatus } from '@quiet/types'
+import { publicChannelsSelectors } from '../../publicChannels/publicChannels.selectors'
+import { messagesSelectors } from '../messages.selectors'
+import { usersSelectors } from '../../users/users.selectors'
+import { communitiesSelectors } from '../../communities/communities.selectors'
 
 export function* verifyMessagesSaga(
   action: PayloadAction<ReturnType<typeof messagesActions.incomingMessages>>['payload']
 ): Generator {
   const messages = action.payload.messages
 
   for (const message of messages) {
+    const generalChannel = yield* select(publicChannelsSelectors.generalChannel)
+    if (!generalChannel) return
```

Comment:
Doesn't it stop executing for loop for channels other than 'general'? I think we do want to verify messages incoming in other channels as well